openapi/src/parts/workspace_cloudcontext.yaml

```diff
@@ -84,6 +93,15 @@ components:
           description: The ID of Azure resource group associated with the workspace.
           type: string
 
```

Comment:
I can get this plumbed through...  Is there anything else we want to include from the stored AWS cloud context metadata?:

https://github.com/DataBiosphere/terra-workspace-manager/blob/jczerk/aws_wlz_interface/service/src/main/java/bio/terra/workspace/service/workspace/model/AwsCloudContext.java#L16

I'm thinking most of it is probably internal for now, and we just want to return something to indicate that we have an AWS context.  Eventually we'd probably want this to be some Workspace Landing Zone identifier.

Reply:
Landing zone Id makes more sense than the account number here. Updating

Reply:
in GCP context we display the projectId as part of workspace description. landingZone is unlikely familiar to the user. And they may not have access to the account for login either.

Reply:
👍